src/daemon/collector/content/collect.js

```diff
@@ -103,9 +103,10 @@ const hideMenu = () => {
   for (let i=0; i<10; i++){
     setTimeout(function () {
       const menus = document.querySelectorAll('[role="menu"]')
+      const areAreasExpanded = document.querySelectorAll('[aria-expanded="false"][aria-haspopup="menu"]');
       if (menus) {
         for (let j=0; j<menus.length; j++) {
-          if (menus[j].clientHeight > 0 && !menus[j].closest('[data-pagelet="ChatTab"]') && menus[j].innerText.indexOf('Notfification') === -1) {
+          if (menus[j].clientHeight > 0 && !menus[j].closest('[data-pagelet="ChatTab"]') && menus[j].innerText.indexOf('Notfification') === -1 && !areAreasExpanded[j] && !areAreasExpanded[j].innerHTML) {
```

Comment:
- Can we add a carriage return somewhere in here to make this line more readable.
- Can we wrap the `!areAreasExpanded[j] && !areAreasExpanded[j].innerHTML` into something like `isUserClickedMenu()` to make this more readable.

Thanks